### PR TITLE
Update features for 1.17, fix effect names for 1.17

### PR DESCRIPTION
**Pull request by Archengius**

### CODE REVIEW

**Comment by rom1504** [comment]
failing

File: lib/features.json [hunk 1]
```diff
@@ -17,11 +17,16 @@
...
   {
     "name": "climbUsingJump",
     "description": "Entity can climb ladders and vines by pressing jump",
-    "versions": ["1.14", "1.15", "1.16"]
+    "versions": ["1.14", "1.15", "1.17"]
+  },
+  {
+    "name": "effectNamesAreRegistryNames",
+    "description": "Status effect names equal to their registry names",
+    "versions": ["1.17", "1.17"]
```
Comment (rom1504) [inline lib/features.json:30]: duplicated

Comment (Archengius) [reply lib/features.json:30]: isn't it [minVersion, maxVersion] like in mineflayer?

**Comment by rom1504** [comment]
would be great if you could review this one @Karang

File: lib/features.json (hunk 1 above)
Comment (rom1504) [reply lib/features.json:30]: I don't think so, but btw we should put this support feature thing in node-minecraft-data so it's the same everywhere

Comment (Archengius) [reply lib/features.json:30]: alright, removed the duplicate

**Review by rom1504 — APPROVED** [review]

**Pull request merged**